projects/storefrontlib/src/cms-components/asm/customer-selection/customer-selection.component.ts

```diff
@@ -79,4 +90,12 @@ export class CustomerSelectionComponent implements OnInit, OnDestroy {
   ngOnDestroy(): void {
     this.subscription.unsubscribe();
   }
+
+  onDocumentClick(event) {
+    if (!!this.resultList) {
```

Comment:
I think it's best to change it to the Boolean constructor for better readability

Reply:
Fixed